spec/Appendix A -- Notation Conventions.md

```diff
@@ -216,3 +216,38 @@ Fibonacci(number):
 Note: Algorithms described in this document are written to be easy to
 understand. Implementers are encouraged to include equivalent but optimized
 implementations.
+
+## Data Collections
+
+A data collection is a finite, iterable aggregation of elements whose iteration
```

Comment:
Why 'aggregation'?

- collection: tautology - "a data collection is a collection"
- set: we define "set" below, so using set here would introduce ambiguity
- group: evokes group theory, best avoided
- assembly: technically correct, but feels unnatural

Reply:
I removed this in editorial.

My thinking is that any reader of this spec does not need to learn what a data collection is in general (though fwiw, wikipedia does use the term "group"). 

What is important for us to clearly state is what aspects of that data collection is normative